Sources/Apollo/MultipartFormData.swift

```diff
@@ -61,7 +61,7 @@ class MultipartFormData {
                     filename: filename)
   }
 
-  func appendPart(inputStream: InputStream,
+  public func appendPart(inputStream: InputStream,
```

Comment:
If this needs to be public, can you please add documentation?

Reply:
Pushed. Please check for typos. My English is not the best 😅